mlem/contrib/requirements.py

```diff
@@ -0,0 +1,45 @@
+import logging
+import os
+from typing import ClassVar, Optional
+
+from mlem.core.objects import MlemBuilder, MlemModel
+from mlem.core.requirements import Requirements
+from mlem.ui import EMOJI_OK, EMOJI_PACK, echo
+
+REQUIREMENTS = "requirements.txt"
+
+logger = logging.getLogger(__name__)
+
+
+class RequirementsBuilder(MlemBuilder):
+    """MlemBuilder implementation for building requirements"""
+
+    type: ClassVar = "requirements"
+
+    target: Optional[str] = None
+    """Target path for the requirements.txt file"""
+    platform: str = "pip"
```

Comment:
add a validator to ensure that this is in `list_implementations(Requirement)`

Reply:
also, platform is not good choice because there are `file` and `custom` requirements. You can use `req_type` or smth like this

Reply:
changed to `req_type` and added a validator that raises a `ValueError`